src/commands/buildApp.ts

```diff
@@ -96,7 +96,16 @@ export async function buildApplication (node: DeviceNode | OSVerNode | PlatformN
 		await vscode.tasks.executeTask(task);
 	} catch (error) {
 		if (error instanceof InteractionError) {
-			await handleInteractionError(error);
+			await handleInteractionError(error)
+				.then(async function () {
+					const choice = await vscode.window.showErrorMessage('Build App failed', { title: 'Rerun' });
+					if (!choice) {
+						return;
+					}
+					if (choice.title === 'Rerun') {
```

Comment:
<!--
  0 failure: 
  1 warning:  src/commands/buil...
  
  
  DangerID: danger-id-Danger;
  File: src/commands/buildApp.ts;
  Line: 105;
-->

- :warning: src/commands/buildApp.ts line 105 – Each then() should return a value or throw (promise/always-return)